app/controllers/evidence_controller.rb

```diff
@@ -91,6 +95,14 @@ def income_form_save
     end
   end
 
+  def record_confirmation
+    evidence.update(
+      completed_at: Time.zone.now,
+      completed_by_id: current_user.id
```

Comment:
I think we should setup `belongs_to` relationship in `EvidenceCheck` and then you can just pass the `current_user`, instead of directly manipulating id.